README.md

```diff
@@ -1,2 +1,172 @@
-# sigtools_workflow
-Workflow for the sigtools and HRdetect R packages 
+# sigTooler
+
+Homologous Recombination Deficiency (HRD) Prediction Workflow using sig.tools
+
+## Overview
+
+## Dependencies
+
+* [gatk 4.2.0.0](https://github.com/broadinstitute/gatk/releases)
+* [tabix 1.9](http://www.htslib.org/doc/tabix.html)
+* [bcftools 1.9](https://samtools.github.io/bcftools/bcftools.html)
+* [sigtools 0.0.0.9000](https://github.com/Nik-Zainal-Group/signature.tools.lib)
+* [bis-rlibs 0.1](https://ggplot2.tidyverse.org/)
+
+
+## Usage
+
+### Cromwell
+```
+java -jar cromwell.jar run sigTooler.wdl --inputs inputs.json
+```
+
+### Inputs
+
+#### Required workflow parameters:
+Parameter|Value|Description
+---|---|---
+`structuralVcfFile`|File|Input VCF file of structural variants (eg. from delly)
+`smallsVcfFile`|File|Input VCF file of SNV and indels (small mutations) (eg. from mutect2)
+`smallsVcfIndex`|File|Index of input VCF file of SNV and indels
+`segFile`|File|File for segmentations, used to estimate number of segments in Loss of heterozygosity (LOH) (eg. from sequenza)
+`indelVAF`|String|Variant Allele Frequency for filtering of indel mutations
+`snvVAF`|String|Variant Allele Frequency for filtering of SNVs
+`tissue`|String|Cancerous-tissue of origin
+`rScript`|String|Temporary variable to call the .R script containing sigtools, will be modulated. default: ~/sigtools_workflow/sigTools_runthrough.R
```

Comment:
Once the script is in Modulator, it will be on the execution path. Rather than committing a temporary variable you have to take out later, it might be easier to just put the temporary script on the path:
`
export PATH=$SCRIPT_DIR:$PATH
`
Not crucial to change it this time around, just something to keep in mind.